index.html

```diff
@@ -22,17 +22,146 @@
             <section>
                 <h1 id="greeting">Hey There. I'm Hans</h1>
                 <h2 id="whoami">I am a software developer</h2>
-                <p>I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                <p id="par1">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
                 </p>
             </section>
-            <ul>
-                <li><img src="images/cat.png" alt="cat"></li>
-                <li><img src="images/instagram.png" alt="instagram"></li>
-                <li><img src="images/hand.png" alt="hand"></li>
-                <li><img src="images/twitter.png" alt="twitter"></li>
-                <li><img src="images/eme.png" alt="eme"></li>
+            <ul class="images1">
+                <li><img src="images/cat.png" alt="cat" class="img1"></li>
+                <li><img src="images/instagram.png" alt="instagram" class="img1"></li>
+                <li><img src="images/hand.png" alt="hand" class="img1"></li>
+                <li><img src="images/twitter.png" alt="twitter" class="img1"></li>
+                <li><img src="images/eme.png" alt="eme" class="img1"></li>
             </ul>
         </div>
+
+        <div class="second-container">
+            <h2 id="recent">My Recent Works</h2>
+        </div>
+
+        <div class="third-container">
+            <img src="images/placeholder.png" alt="placeholder" class="img2">
+            <h3 id="multi">Multi-Post Sories</h3>
+            <p id="par2">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <nav class="langs">
+                <button class="langbuttons">css</button>
+                <button class="langbuttons">html</button>
+                <button class="langbuttons">bootstrap</button>
+                <button class="langbuttons">Ruby</button>
+            </nav>
```

Comment:
Kindly use a list `ul` and each of the project technologies (e.g., "JavasScript", "HTML", and "CSS")  should be a `li` element because it is semantic that way. The `nav` element should only be used for the `navbar` menu items since they are used for `navigation`, hence `nav`.

Reply:
I implemented ul and li where requested, but I have a little problem. The elements tend to go more to the right. I think it is because of the bullet point. Please if you can help me to correct this I will be grateful.

Reply:
You can use a `padding: 0;` on the parent `ul` tag to remove the default indentation of `li` items.